pom.xml

```diff
@@ -1641,7 +1641,7 @@
                 <plugin>
                     <groupId>ca.vanzyl.provisio.maven.plugins</groupId>
                     <artifactId>provisio-maven-plugin</artifactId>
-                    <version>1.0.9</version>
+                    <version>1.0.16</version>
```

Comment:
Will the upstream change still be part of 16? Or would it require the next version.